std_msgs/Header header
float32 load_cell_0
float32 load_cell_1
float32 load_cell_2

